### PR TITLE
Edit getting started docs

### PR DESCRIPTION
**Pull request by sophia-wiley**
Changed word to match UI text.

### CODE REVIEW

File: docs/cloud/getting-started-with-airbyte-cloud.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ A destination is a data warehouse, data lake, database, or an analytics tool whe
 To set up a destination:
 
 1. On the Airbyte Cloud dashboard, click **Destinations** in the left navigation bar. 
-2. In the top right corner, click **+ new destination**.
+2. In the top right corner, click **+ New destination**.
```
Comment (sophia-wiley) [inline docs/cloud/getting-started-with-airbyte-cloud.md:46]:
```suggestion
2. In the top right corner, click **+ New destination**.
```

File: docs/readme.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Here's a list of our [connectors with their health status](integrations/README.m
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
 docker-compose up
-```
+```
```
Comment (supertopher) [inline docs/readme.md:25]:
```suggestion
```
```

**Review by supertopher — APPROVED** [review]
commit my change and then ship it

File: docs/readme.md [hunk 3]
```diff
@@ -22,7 +22,6 @@ Here's a list of our [connectors with their health status](integrations/README.m
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
 docker-compose up
```
Comment (supertopher) [inline docs/readme.md:24]:
```suggestion
``` 
```

**Pull request merged**